conf/machine/rpi4-omnect-lab.conf

```diff
@@ -0,0 +1,28 @@
+# this is an artificial omnect product for testing purposes of features like
+# LTE or TPM which is basically a plain Raspberrypi 4 with a USB LTE modem and
+# a TPM module
+
+require conf/machine/raspberrypi4-64.conf
+require conf/machine/raspberrypi4-64.tpm.conf
+
+require conf/machine/include/rpi.inc
+
+DISTRO_FEATURES:append = " 3g tpm2"
+MACHINE_FEATURES:append = " 3g tpm2"
+
+# adu compatibility id
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "0"
+
+# wic: how to partition
+WKS_FILE = "omnect-os.rpi4.wks.in"
+
+# u-boot boot cmd config vars
+UBOOT_ARCH = "arm64"
+
+# no differentiation between release an devel images concerning rootfs size
+OMNECT_PART_SIZE_ROOTFS    ?= "${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '1', '900000', '900000', d)}"
+
+# LTE modem is expected on lower USB3 port, and supported LTE modems are ...
+# - Sierra Wireless Airprime EM7455 (wwan interface wwp1s0u1u2i8), or
+# - Quectel EC25-EUX (wwan interface wwp1s0u1u2i8)
+OMNECT_ONLINE_INTERFACES = "eth0:${OMNECT_WLAN0}:wwp1s0u1u2i8:wwp1s0u1u2i4"
```

Comment:
OMNECT_WLAN0 is also used in wpa-supplicant_%.bbappend, which means it makes sense to me to keep that as it is, because the variable links these two things.

Reply:
yes, but you can change it to `wlan0` there as well.

Reply:
sure, and what's the point in removing the OMNECT_WLAN0 which made clear that wlan0 interface is used in several places, so that it cannot be forgotten in case a device with a different WLAN interface is to be supported someday in the future?
adding the variable in the first place was for a reason I guess, and that is as I would have done it, too.

Reply:
what's the point of not using a variable for wwan interfaces then?

Reply:
you requested - for clarity, which I could follow - to use explicit values for default online interfaces in one place instead of listing the already existing OMNECT_ETHx variables, so I did the same for the wwan interfaces.
both kinds of interfaces aren't used elsewhere separately, so this is ok for me.
but the wlan interface is different, so the "abstraction" by a variable makes sense.
defining variables just to have them (as it was back then with the ethernet variables) also doesn't make sense to me, but that's different here, isn't it?